src/RequestForm.js

```diff
@@ -258,6 +265,37 @@ class RequestForm extends React.Component {
         this.setState({ blocked: true });
       }
     }
+
+    if (prevParentResources?.configs?.records[0]?.value !== parentResources?.configs?.records[0]?.value) {
+      const {
+        titleLevelRequestsFeatureEnabled: newTitleLevelRequestsFeatureEnabled,
```

Comment:
@Stanislau-Snapok is there any reason you are renaming to `titleLevelRequestsFeatureEnabled` to `newTitleLevelRequestsFeatureEnabled`?

Without renaming you could just do:

````js
this.setState(
  {
    titleLevelRequestsFeatureEnabled,
  },
  this.setTlrCheckboxInitialState(),
);
````

Reply:
IDK how it's happend :) probably there was another new logic which was removed by me as unnecessary in the process of implementation.
Done. Thanks!